Core/GDCore/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -136,6 +136,20 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSceneExtension(
       .AddCodeOnlyParameter("currentScene", "")
       .AddParameter("yesorno", _("Deactivate input when focus is lost"))
       .MarkAsAdvanced();
+
+  extension
+      .AddCondition(
+          "HasGameJustResumed",
+          _("Game resumed"),
+          _("Check if the game has just resumed (from being hidden). It "
```

Comment:
I think the rest of the description is clear enough (a tab is selected, minimized window on desktop or app put on front on mobile)

```suggestion
          _("Check if the game has just resumed. It "
```

Reply:
Tristan suggested to just drop the parenthesis (but keep the content). I think it's a good call. It will avoid users to think it's a "scene resumed" condition if they read fast.